src/runtime-rs/crates/hypervisor/src/dragonball/inner_hypervisor.rs

```diff
@@ -14,7 +14,7 @@ use kata_types::capabilities::Capabilities;
 
 use super::inner::DragonballInner;
 use crate::{utils, VcpuThreadIds, VmmState};
-use persist::KATA_PATH;
+use shim_services::KATA_PATH;
```

Comment:
NIT: It's kinda strange that `inner_hypervisor` use `shim_service`, could you find some better space to put `KATA_PATH`?

Reply:
I agree, but there would be another problem.
1. this variable has to reside in a library.
2. if the variable is not in shim_services, then shim_services will introduce another library as dependency.
I think the library should not be dependent on any kata libararies? Maybe this could be resolved in another PR?

Reply:
I have adopted @jodh-intel 's advice, use another name as `shim-interface`, as further interfaces facing other binaries could be put their. Variables like `KATA_PATH` makes more sense as well.